SalesforceCommerceCloud/pwa-kit
# Remove react-query-devtools from production build

# Description

Goal: remove `react-query-devtools` from production build to reduce bundle size.

Changes:
- removes `react-query-devtools` from `commerce-sdk-react`
- install `react-query-devtools` in retail react app
- Lower bundle size test thresholds.

### Bundle analysis before & after

BEFORE

<img width="1710" alt="Screenshot 2023-04-10 at 11 41 02 AM" src="https://user-images.githubusercontent.com/10948652/230972494-a0acf9ab-f9da-4450-b12c-da45c4e9a260.png">

AFTER

<img width="1711" alt="Screenshot 2023-04-10 at 11 38 16 AM" src="https://user-images.githubusercontent.com/10948652/230972530-7b890501-9002-4974-9faa-ab9618cfa5f4.png">




# Types of Changes




# How to Test-Drive This PR

- run `npm start` in ./packages/template-retail-react-app and open localhost:3000 and notice the devtool icon on left bottom corner
- run `NODE_ENV=production npm start`, verify the icon is gone
- run `npm run analyze-build` and verify the `react-query-devtools` is not in the build
